pr_check.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
We had already using GH Actions for PR checks. This would be redundant for us. 

I guess you need to run only the CD, not CI.

CC: @prashbnair